packages/react-ui-components/src/SelectBox_ListPreview/selectBox_ListPreview.js

```diff
@@ -66,9 +68,9 @@ export default class SelectBox_ListPreview extends PureComponent {
                             />
                     </div>
                 )}
-                <div className={theme.selectBox__item}>
-                    <SelectBox_CreateNew {...this.props}/>
-                </div>
+                { createNew ? <div className={theme.selectBox__item}>
```

Comment:
This smell bad, but I don't want to have an empty element at the end of the list. Break last-child, and make styling quiet hacky